### PR TITLE
Use custom releaser image to deploy hac application

### PR DESCRIPTION
**Pull request by karelhala**
### Add custom releaser image

We want to use ssh with our image, so let's use https://quay.io/repository/redhat-cloud-services/releaser

### Remove release image

Since we can't really use docker-in-docker let's remove this step for now. We'll enable image release with https://docs.ci.openshift.org/docs/architecture/ci-operator/#building-container-images

### CODE REVIEW

**Comment by karelhala** [comment]
The failing test is because `coreos-rpm-ostree-main-presubmits` is outdated. Should I wait for the team to fix it or push it myself?

cc @andrewballantyne 

/retest

**Comment by andrewballantyne** [comment]
> The failing test is because `coreos-rpm-ostree-main-presubmits` is outdated. Should I wait for the team to fix it or push it myself?

@karelhala I'm not sure I follow... what would you need to do to fix it? This appears to be a release outdated issue? Or like something HAC?

**Comment by karelhala** [comment]
@andrewballantyne yup, this is release outdated issue. I could include the file, but I don't think that's a good approach.

**Review by andrewballantyne — APPROVED** [review]

**Comment by karelhala** [comment]
ping @petr-muller can you please review? Thank you!

File: core-services/image-mirroring/supplemental-ci-images/mapping_simplemental_ci_images_redhat_cloud_services [hunk 1]
```diff
@@ -0,0 +1 @@
+quay.io/redhat-cloud-services/releaser:latest registry.ci.openshift.org/redhat-cloud-services/releaser:latest
```
Comment (petr-muller) [inline core-services/image-mirroring/supplemental-ci-images/mapping_simplemental_ci_images_redhat_cloud_services:1]: please fix the typo in the file name (simplemental -> supplemental)

**Review by petr-muller — APPROVED** [review]

**Pull request merged**